src/common/plugins.js

```diff
@@ -107,19 +108,22 @@ class Plugins extends EventEmitter {
     try {
       this.reset()
       this.config = await load(this.configFile)
+      this.config.forEach(config => {
```

Comment:
Is this necessary? Let's use a `disabled` flag instead of `enabled` -- then we probably don't need to set a default value here, right?

Reply:
Right, I just went with the _positive_ word, but disabled is better here.